module/VuFind/src/VuFind/Controller/InstallController.php

```diff
@@ -227,8 +227,8 @@ protected function phpVersionIsNewEnough()
             return false;
         }
 
-        // We need at least PHP v7.3.0:
-        return PHP_VERSION_ID >= 70300;
+        // We need at least PHP version as defined in MINIMAL_PHP_VERSION:
```

Comment:
This comment seems to suggest that there is a constant called MINIMAL_PHP_VERSION; maybe it should be revised.

Reply:
Fixed, now it references composer.json file.